modules/checklist/assets/js/components/checklist-card-content.js

```diff
@@ -34,16 +66,18 @@ const ChecklistCardContent = ( { step } ) => {
 			</CardContent>
 			<CardActions>
 
-				{ isLocked
-					? null
-					: <Button
+				{ shouldShowMarkAsDone
+					? <Button
 							size="small"
 							color="secondary"
 							variant="text"
-							className="mark-as-done"
+							className="mark-as-button"
```

Comment:
suggested mark-as-done-button?

Reply:
I wanted the button to have the same class wether is done or not (for tests), WDYT?

Reply:
@TzviRabinovitch @louiswol94 
WDTY? We were talking about this making the code "dirty" because I only added the selectors to be used by the test. Should I keep it or switch it to some other selector?

Reply:
Removed all selectors!